aspnet/mvc/models/validation/sample/UsersController.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.AspNet.Mvc;
+using MVCMovie.Models;
+
+namespace MVCMovie.Controllers
+{
+public class UsersController : Controller
+{
+    private IUserRepository userRepo;
+    public UsersController()
+    {
+        this.userRepo = new UserRepository();
+    }
+
+    [AcceptVerbs("Get", "Post")]
+    public IActionResult VerifyEmail(string email)
+    {
+        if (!this.userRepo.VerifyEmail())
+        {
+            return Json(data: string.Format("Email {0} is already in use.", email));
+        }
+
+        return Json(new { success = true });
```

Comment:
I'm pretty sure this needs to be `Json(data: true)` i.e. return just the value and not an object containing it.